src/include/duckdb/function/aggregate_function.hpp

```diff
@@ -15,6 +15,14 @@
 
 namespace duckdb {
 
+//! The half-open range of frame boundary values relative to the current row
+struct FrameStats {
```

Comment:
I wonder if this is the best place to put this struct. From my understanding, frames are a concept that is very specific to the windowing functionality, and not to aggregation in general.

Reply:
It put it there because the windowing APIs use it and they are defined in that file. Prying apart window and general aggregation is unfortunately rather difficult.